modules/osx_defaults.bash

```diff
@@ -108,7 +108,6 @@ defaults write com.apple.dock persistent-apps -array
 for app in \
   System\ Preferences \
   Safari \
-  HipChat \
```

Comment:
May be replace it with Slack icon, not just remove? Can you pls add Slack instead of Hipchat and test this module at your laptop?